app/services/external_api/pacman_service.rb

```diff
@@ -0,0 +1,149 @@
+# frozen_string_literal: true
+
+require "json"
+require "base64"
+require "digest"
+
+class ExternalApi::PacmanService
+  BASE_URL = ENV["PACMAN_API_URL"]
+  SEND_DISTRIBUTION_ENDPOINT = "/package-manager-service/distribution"
+  SEND_PACKAGE_ENDPOINT = "/package-manager-service/communication-package"
+  GET_DISTRIBUTION_ENDPOINT = "/package-manager-service/distribution/"
+  HEADERS = {
+    "Content-Type": "application/json", Accept: "application/json"
+  }.freeze
+
+  class << self
+    # Purpose: Creates and sends communication package
+    # POST: /package-manager-service/communication-package
+    #
+    # takes in file_number(string), name(string), document_reference(json of strings)
+    #
+    # Response: JSON of created package from Pacman API
+    # Example response can be seen in lib/fakes/pacman_service.rb under 'fake_package_request' method
+    def send_communication_package_request(file_number, name, document_references)
+      request = package_request(file_number, name, document_references)
+      send_pacman_request(request)
+    end
+
+    # Purpose: Creates and sends distribution
+    # POST: /package-manager-service/distribution
+    #
+    # takes in package_id(string), recipient(json of strings), destinations(json of strings)
+    #
+    # Response: JSON of created distribution from Pacman API
+    # Example response can be seen in lib/fakes/pacman_service.rb under 'fake_distribution_request' method
+    def send_distribution_request(package_id, recipient, destinations)
+      request = distribution_request(package_id, recipient, destinations)
+      send_pacman_request(request)
+    end
+
+    # Purpose: Gets distribution from distribution id
+    # POST: /package-manager-service/distribution
+    #
+    # takes in distribution_id(string)
+    #
+    # Response: JSON of distribution from Pacman API
+    # Example response can be seen in lib/fakes/pacman_service.rb under 'fake_distribution_response' method
+    def get_distribution_request(distribution_id)
+      request = {
+        endpoint: GET_DISTRIBUTION_ENDPOINT + distribution_id, method: :get
+      }
+      send_pacman_request(request)
+    end
+
+    private
+
+    # Purpose: Builds package request
+    #
+    # takes in file_number(string), name(string), document_reference(json of strings)
+    #
+    # Response: package request hash
+    def package_request(file_number, name, document_reference)
+      request = {
+        body: {
+          fileNumber: file_number,
+          name: name,
+          documentReferences: {
+            id: document_reference[:id],
+            copies: document_reference[:copies]
+          }
```

Comment:
```suggestion
          documentReferences: [
            {
              id: document_reference[:id],
              copies: document_reference[:copies]
            }
          ]
```
Pacman expects `documentReferences` to be an array.